ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/snmp-settings-2.yml

```diff
@@ -14,6 +14,7 @@ l2leaf:
 default_mgmt_method: inband
 
 snmp_settings:
+  enable_inband_mgmt_vrf: true
```

Comment:
we don't want this in eos_designs_unit_tests as it is being deprecated. For now it looks not covered because of this: https://github.com/aristanetworks/avd/issues/5544